libs/gdk/src/lib/gdk-card/gdk-card.component.scss

```diff
@@ -28,8 +28,8 @@ gdk-card .mat-card-content {
   }
   .mat-list[role='list'] {
     display: grid;
-    grid-row-gap: 10px;
-    grid-auto-rows: 60px;
+    grid-row-gap: 20px;
+    grid-auto-rows: min-content;
```

Comment:
Maybe is better to search a better solution due to lack of compatibility, what do you think? - https://caniuse.com/#search=min-content

Reply:
I changed to auto, but testing with a more than 2 lines hint I realize that both options didn't fixed the problem. So i changed some positions too.